stores/history.ts

```diff
@@ -0,0 +1,56 @@
+import { atom } from 'nanostores'
+
+import { debounce } from '../lib/time.js'
+import { current, type LchValue } from './current.js'
+
+let $undos = atom<LchValue[]>([])
```

Comment:
Do we have `$` in store name in other stores?

If not it is better to keep origin name system.

We can switch to `$` in separated PR. I like this idea.

Reply:
done. I understand it and it was that before. but with prev, I few times was confused is it a variable or a store. and in documentation I saw this $ sign. then I left with undos store and $prev store

Reply:
If you have time, let's migrate to `$` when we will finish with this PR